### ISSUE TITLE
Crossroads Caravan Pickup (Quest 13949)

### ISSUE THREAD

**Issue by CoriolanusSnow** [TrinityCore/TrinityCore#20059]
**Description:**

The Player must ride for this Quest, but the player can not ride the animal.

**Current behaviour:**

The Player becomes Halgas "Street Sweeper" and must escort the Caravan, but can't ride.

**Expected behaviour:**

The Player becomes Halgas "Street Sweeper" and escort the Caravan and ride on this Komdo. And can then shot with the item.

**Steps to reproduce the problem:**

1. Start the Quest by Halga Bloodeye
2. Try to ride

**Branch(es):**

master

**TC rev. hash/commit:** 

TrinityCore rev. 33478689bb1b+ 2017-07-18 18:19:07 +0200 (master branch) (Win64, Release, Static) (worldserver-daemon)

**TDB version:** 720.00

**Operating system:** Windows Server 2012 R2

**Comment by buddyqc69** [comment]
Do you have fix for this quest ?

**Comment by illfated** [comment]
No, there does not appear to be any suggestions on how to fix this quest yet. Feel free to suggest one, if you know more about it. Not sure if this is a vehicle seat issue or something else.

**Comment by infidelus** [comment]
After accepting this quest you are directed to speak to Halga Bloodeye to start the escort quest but are unable to speak to her again.

[Quest](https://www.wowhead.com/quest=13949/crossroads-caravan-pickup)

**Comment by aquadeus** [comment]
this quest is complex to script properly
